application/log_handlers/handlers.py

```
@@ -0,0 +1,128 @@
+    """
+    Extends `CLPFileHandler` to support file size-based log rotation.
+    """
+
+    def __init__(
+            self,
+            filename_prefix: str,
+            log_dir: Path,
+            max_bytes: int,
+            backup_count: int = 20,
+            mode: str = "ab",
+            enable_compression: bool = True,
```

Ruff (0.8.2)

Boolean-typed positional argument in function definition

(FBT001)